docs/routing/index.rst

```diff
@@ -8,13 +8,23 @@ Routing
    :maxdepth: 1
 
    arp
+   bfd
    bgp
-   rpki
+   mpls
+   mss-clamp
+   multicast
    ospf
    pbr
    rip
-   static
-   bfd
-   mss-clamp
    routing-policy
-   multicast
+   rpki
+   static
+   
```

Comment:
Why so many empty lines?

Reply:
I tried to sort them out alphabetically.
Where are there empty lines?

Reply:
Do you mean here in docs/routing/index.rst?

Reply:
Found the empty lines! I updated it.